src/packages/core/components/input-tiny-mce/input-tiny-mce.element.ts

```diff
@@ -42,6 +42,19 @@ export class UmbInputTinyMceElement extends FormControlMixin(UmbLitElement) {
 		return this._editorElement?.querySelector('iframe') ?? undefined;
 	}
 
+	set value(newValue: FormDataEntryValue | FormData) {
+		this._value = newValue;
```

Comment:
Sadly we dont have any way to enforce calling the super method. But the FormControlMixin has a getter and setter for the value that we need to call.

So this should be changed into: `super.value = newValue`

Reply:
Otherwise really good! 👍

Reply:
@nielslyngsoe So you are saying that 

```
this._value = newValue;
```

should be

```
super.value = newValue;
```

I'll try that and update the PR =D

Reply:
Updated with new commit